huxley/api/views/school.py

```diff
@@ -41,6 +40,13 @@ def get_queryset(self):
 
         return Assignment.objects.filter(school_id=school_id)
 
+class SchoolFinalize(generics.CreateAPIView):
```

Comment:
Doesn't make much sense to use a CreateAPIView here, since that's specifically designed for creating resources, e.g. models. Instead, you can just extend from `GenericAPIView` (I believe) and leave everything else the same.